### PR TITLE
do not use branch name in db name

### PR DESCRIPTION
**Pull request by neerajdotname**

### CODE REVIEW

File: config/database.yml.postgresqlapp [hunk 1]
```diff
@@ -4,12 +4,15 @@
 # Check doc/why_database_name_only_63_characters_long.md to see
 # details about the restriction on database name
 
+# If you prefer to have branch name in the database name then use following
+# database: <%= "#{repository_name}_development_#{branch_name}"[0...63] %>
+
 development:
   adapter: postgresql
-  database: <%= "#{repository_name}_development_#{branch_name}"[0...63] %>
+  database: <%= "#{repository_name}_development"[0...63] %>
```
Comment (vipulnsward) [inline config/database.yml.postgresqlapp]: Need to remove `[0..63]`

**Comment by vipulnsward** [comment]
:+1: I end up removing this in every project.

File: config/database.yml.postgresqlapp (hunk 1 above)
Comment (mechanicles) [inline config/database.yml.postgresqlapp]: Yup. I also don't use it. I keep it simple.

Comment (neerajdotname) [inline config/database.yml.postgresqlapp]: I would suggest to keep it. It does not do any harm and if the repo name is long then it fixes thing.

Comment (vipulnsward) [inline config/database.yml.postgresqlapp]: @neerajdotname yes, realised that. Its merged now.

**Pull request merged**